src/fabric.c

```diff
@@ -917,6 +923,11 @@ void fi_ini(void)
 
 	ofi_register_provider(COLL_INIT, NULL);
 
+	ret = pthread_atfork(NULL, NULL, ofi_child_atfork_handler);
```

Comment:
This breaks windows build.  Why the indirection, versus calling ofi_memhooks_atfork_handler() directly?

Reply:
Will look into build issue. I'll remove the indirection.